src/PrestaShopBundle/Resources/config/services/core/form/form_data_provider.yml

```diff
@@ -106,6 +106,11 @@ services:
       - '@=service("prestashop.adapter.shop.context").getShops(true, true)'
       - '@=service("prestashop.adapter.legacy.context").getContext().country.id'
 
+  PrestaShop\PrestaShop\Core\Form\IdentifiableObject\DataProvider\AttributeGroupFormDataProvider:
+    autowire: true
```

Comment:
```suggestion
    autowire: true
    public: false
```

Reply:
Indeed as much as possible we should now define services as private by default, only when they absolutely need to be called via `Container::get` should be allow them to be public (even then we should think about how to avoid this kind of usage as much as possible in the future)